src/storage/sdr/record/fru_device_locator.rs

```diff
@@ -55,7 +55,8 @@ impl FruDeviceLocator {
             })
         };
 
-        let lun = LogicalUnit::try_from((record_data[2] >> 3) & 0b11).ok()?;
+        // NOTE(unwrap): `LogicalUnit::try_from` always succeeds for data masked with 0b11.
+        let lun = LogicalUnit::try_from((record_data[2] >> 3) & 0b11).unwrap();
```

Comment:
How would you feel about moving the bitmask into a `LogicalUnit::from_low_bits(field:u8)` and doing the unwrap there?

Reply:
I'm a fan! Have added it